theme/azul.js

```diff
@@ -0,0 +1,209 @@
+
+        color: colorPalette,
+
+        title: {
+            textStyle: {
+                fontWeight: 'normal',
+				color: '#f2385a'
+            }
+        },
+
+        visualMap: {
+            color:['#f2385a','#f5a503']
+        }, 
+
+        toolbox: {
+            color : ['#f2385a','#f2385a','#f2385a','#f2385a']
+        },
+
+    tooltip: {
+        backgroundColor: 'rgba(0,0,0,0.5)',
+        axisPointer : {            // Axis indicator, coordinate trigger effective
+            type : 'line',         // The default is a straight line： 'line' | 'shadow'
+            lineStyle : {          // Straight line indicator style settings
+                color: '#f2385a',
+                type: 'dashed'
+            },
+            crossStyle: {
+                color: '#f2385a'
+            },
+            shadowStyle : {                     // Shadow indicator style settings
+                color: 'rgba(200,200,200,0.3)'
+            }
+        }
+    },
+
+    // Area scaling controller
+    dataZoom: {
+        dataBackgroundColor: '#eee',            // Data background color
+        fillerColor: 'rgba(200,200,200,0.2)',   // Fill the color
+        handleColor: '#f2385a'     // Handle color
+    },
+
+    timeline : {
+        lineStyle : {
+            color : '#f2385a'
+        },
+        controlStyle : {
+            normal : { color : '#f2385a'},
+            emphasis : { color : '#f2385a'}
+        }
+    },
+
+	k: {
+        itemStyle: {
+            normal: {
+                color: '#f5a503',          // Yang line filled with color
+                color0: '#e6efdc',      // Yinxian fill color
+                lineStyle: {
+                    width: 1,
+                    color: '#e6efdc',   // Yang line border color
+                    color0: '#f5a503'   // Yinbian border color
+                }
+            }
+        }
+    },
+
+        candlestick: {
+            itemStyle: {
+                normal: {
+                    color: '#f2385a',
+                    color0: '#f5a503',
+                    lineStyle: {
+                        width: 1,
+                        color: '#f2385a',
+                        color0: '#f5a503'
+                    }
+                }
+            }
+        },
+
+        graph: {
+            color: colorPalette
+        },
+
+        map: {
+        itemStyle: {
+            normal: {
+                areaStyle: {
+                    color: '#ddd'
+                },
+                label: {
+                    textStyle: {
+                        color: '#c12e34'
+                    }
+                }
+            },
+            emphasis: {                 // Is also selected style
+                areaStyle: {
+                    color: '#e6efdc'
+                },
+                label: {
+                    textStyle: {
+                        color: '#c12e34'
+                    }
+                }
+            }
+        }
+    },
+
+	force : { 
```

Comment:
`force` has been ranamed to `graph` since 3.0.0

Reply:
Ok, see post on main thread. Thanks!